middleware/throttle.go

```diff
@@ -34,21 +54,46 @@ type token struct{}
 
 // throttler limits number of currently processed requests at a time.
 type throttle struct {
```

Comment:
nothing, I am okay with throttler too, but rename the file too if you guys think its better. I consider that "throttle" is an action on a request.. and a "throttler" is a thing that throttles requests, so, depends on context and what sounds better.. I dunno :P both work